src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/page/FindPatientPage.java

```diff
@@ -0,0 +1,48 @@
+package org.openmrs.contrib.isanteplus.qaframework.automation.page;
+
+import org.openqa.selenium.By;
+
+public class FindPatientPage extends Page {
+	
+	private static final String PAGE_URL = "/coreapps/findpatient/findPatient.page?app=coreapps.findPatient";
+	private static final By PATIENT_SEARCH = By.id("patient-search");
+	private static final By PATIENT_ID_SEARCH_RESULT = By.cssSelector("#patient-search-results-table tr:first-child td:first-child");	
+	private static final By PATIENT_NAME_SEARCH_RESULT = By.cssSelector("#patient-search-results-table tbody tr:first-child td:nth-child(2)");
+	
+	public FindPatientPage(Page page) {
+		super(page);
+	}
+
+	public void enterPatientName(String patientName) {
+		setTextToFieldNoEnter(PATIENT_SEARCH, patientName);
+	}
+
+	public  ClinicianFacingPatientDashboardPage clickOnFirstPatient() {
+		clickOn(PATIENT_NAME_SEARCH_RESULT);
+		return new ClinicianFacingPatientDashboardPage(this);
+	}
+
+	public void search(String text) {
+		setTextToFieldNoEnter(PATIENT_SEARCH, text);
+	}
+
+	/**
+	 * Finds first record from the result table
+	 * 
+	 * @return patient id
+	 */
+	public String getFirstPatientIdentifier() {
+		getFirstPatientName();
+		return findElement(PATIENT_ID_SEARCH_RESULT).getText();
```

Comment:
we already have  an [method](https://github.com/IsantePlus/isanteplus-qaframework/blob/main/src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/page/Page.java#L201) in the parent class that does this same thing

Reply:
Hello @mozzy11 , the method that exists gets text  of an element , but without specifying which element, so I think passing through the exact element its supposed to pick is the right way. is there any other option of getting a **searched patient result**